tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ManualInstrumentation/Configuration/TracerSettings/AgentUriGetIntegration.cs

```diff
@@ -0,0 +1,36 @@
+﻿// <copyright file="AgentUriGetIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System.ComponentModel;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Telemetry;
+using Datadog.Trace.Telemetry.Metrics;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.ManualInstrumentation.Configuration.TracerSettings;
+
+/// <summary>
+/// System.Uri Datadog.Trace.Configuration.TracerSettings::get_AgentUri() calltarget instrumentation
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Datadog.Trace.Manual",
+    TypeName = "Datadog.Trace.Configuration.TracerSettings",
+    MethodName = "get_AgentUri",
+    ReturnTypeName = "System.Uri",
+    ParameterTypeNames = new string[0],
+    MinimumVersion = ManualInstrumentationConstants.MinVersion,
+    MaximumVersion = ManualInstrumentationConstants.MaxVersion,
+    IntegrationName = ManualInstrumentationConstants.IntegrationName)]
+[Browsable(false)]
+[EditorBrowsable(EditorBrowsableState.Never)]
+public class AgentUriGetIntegration
+{
+    internal static CallTargetState OnMethodBegin<TTarget>(TTarget instance)
+    {
+        TelemetryFactory.Metrics.Record(PublicApiUsage.ExporterSettings_AgentUri_Get);
```

Comment:
So this says `PublicApiUsage.ExporterSettings_AgentUri_Get`, but the integration is actually the `TracerSettings.AgentUri` is there a different telemetry metric for that?

Reply:
> is there a different telemetry metric for that

I considered it, but decided it wasn't worth worrying about, and it would be fine to share them 🙂